nbs/15_gsa_gls/20-null_simulations/05_gwas/05_05-qc-samples-het_outliers.ipynb

```diff
@@ -0,0 +1,560 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    het$HET_RATE &lt;- (het$"N.NM." - het$"O.HOM.") / het$"N.NM."
</pre><p>Maybe just a comment to spell out het rate = proportion heterozygotes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Yes, thank you!</p>